allink_core/core/models/base_plugins.py

```diff
@@ -162,17 +157,15 @@ class AllinkBaseAppContentPlugin(CMSPlugin):
     )
     load_more_button_text = models.CharField(
         'Text for "Load .."-Button',
-        help_text=
-        'If left blank, a default text will be used. <br>Note: Should the default text be adjusted site-wide,'
+        help_text='If left blank, a default text will be used.<br>Note: Should the default text be adjusted site-wide,'
```

Comment:
The removal of the space before the <br> generates new migrations for the news app (and probably others too)

Reply:
ok. can you please revert the change on all these multiline help_text? and add # noqa